.docker/Dockerfile.nginx-drupal

```diff
@@ -14,3 +14,8 @@ COPY .docker/sanitize.sh /app/sanitize.sh
 
 RUN /app/sanitize.sh \
   && rm -rf /app/sanitize.sh
+
+RUN apk add gettext
+
+ARG X_FRAME_OPTIONS
+RUN envsubst < /etc/nginx/conf.d/drupal/location_drupal_append_x_frame_options.template > /etc/nginx/conf.d/drupal/location_drupal_append_x_frame_options.conf
```

Comment:
Creates the final nginx config by performing env var substitution across the template file.